pyproject.toml

```diff
@@ -12,15 +12,15 @@ readme = "README.md"
 
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = ">=3.8,<3.13"
 pyarrow = "^11.0.0"
 fsspec = "^2023.1.0"
 gcsfs = "^2023.1.0"
 s3fs = "^2023.1.0"
 pydantic = "^1.10.5"
 pandas = "^2.0.0"
 tqdm = "^4.65.0"
-pinecone-client = "^2.2.2"
+pinecone-client = "3.0.0.dev10"
```

Comment:
```suggestion
pinecone-client = "^3.0.0"
```

Reply:
@igiloh-pinecone we need this before merge, cannot use `"^3.0.0.dev0"` because this will incl. `3.0.0.rc2` which does _not_ include `ServerlessSpec`